bin/node/runtime/src/lib.rs

```diff
@@ -254,10 +256,11 @@ impl pallet_balances::Trait for Runtime {
 parameter_types! {
 	pub const TransactionBaseFee: Balance = 1 * CENTS;
 	pub const TransactionByteFee: Balance = 10 * MILLICENTS;
-	// setting this to zero will disable the weight fee.
-	pub const WeightFeeCoefficient: Balance = 1_000;
+	// In the Substrate node, a weight of 10_000_000 (smallest non-zero weight)
+	// is mapped to 10_000_000 units of fees, hence:
+	pub const WeightFeeCoefficient: Balance = 1;
```

Comment:
Is it bad that our coefficient is 1? It kind of means we cannot go lower than this if we ever want to...

Reply:
This + the "smallest non-zero weight" should be re-evaluated.

Reply:
> Is it bad that our coefficient is 1? It kind of means we cannot go lower than this if we ever want to...

Why not? the current implementation might not allow it but with small tweaks we can do `pub const WeightFeeCoefficient: Percent = Percent::from_parts(50);`

Reply:
yeah, no technical reason, it would just require a new impl